sled-agent/src/probe_manager.rs

```diff
@@ -406,7 +412,20 @@ impl ProbeManagerInner {
 
                 // Ask the sled-agent to stop tracking our datalinks, and then
                 // delete the OPTE ports.
-                self.metrics_queue.untrack_zone_links(&running_zone).await;
+                match self.metrics_queue.untrack_zone_links(&running_zone) {
+                    Ok(_) => debug!(
+                        self.log,
+                        "stopped tracking zone datalinks";
+                        "zone_name" => running_zone.name(),
+                    ),
+                    Err(errors) => error!(
+                        self.log,
+                        "Failed to stop tracking one or more datalinks in the \
+                        zone, some metrics will not be produced";
```

Comment:
Is "some metrics will not be produced" accurate for failing to _untrack_ links?

Reply:
It is not, thanks.